test/integration/targets/rpm_key/tasks/rpm_key.yaml

```diff
@@ -187,6 +187,53 @@
       - result is success
       - result is not changed
 
+- name: Issue 83493 - create fragments directory
+  file:
+    path: /tmp/fragments
+    state: directory
+
+- name: Issue 83493 - download AlmaLinux-9 GPG key
+  get_url:
+    url: https://repo.almalinux.org/almalinux/RPM-GPG-KEY-AlmaLinux-9
+    dest: /tmp/fragments/RPM-GPG-KEY-AlmaLinux-9
+
+- name: Issue 83493 - download PGDG RHEL9 GPG key
+  get_url:
+    url: https://download.postgresql.org/pub/repos/yum/keys/RPM-GPG-KEY-PGDG-AARCH64-RHEL9
+    dest: /tmp/fragments/RPM-GPG-KEY-PGDG-AARCH64-RHEL9
+
+- name: Issue 83493 - Assemble key from fragments
+  ansible.builtin.assemble:
+    src: /tmp/fragments
+    dest: /tmp/KEY_83493
+
+- name: Issue 83493 - Verify key fingerprints. This should fail due to multiple key signatures
+  rpm_key:
+    key: /tmp/KEY_83493
```

Comment:
Thanks for working on this. You should be able to swap this out for https://ansible-ci-files.s3.amazonaws.com/test/integration/targets/rpm_key/linux_signing_key.pub now and remove the few tasks preceding this one.